app/views/users/show.html.erb

```diff
@@ -7,7 +12,7 @@
   <strong>Bio:</strong>
   <%= @user.bio %>
 </p>
-<%= image_tag(@user.avatar_url) if @user.avatar? %>
+<div id="avatar"><%= image_tag(@user.avatar_url) if @user.avatar? %></div>
 
```

Comment:
If you incorporate `Hound` into your app, it will complain about unnecessary blank lines.  As you go along, you'll learn what is preferred - they may be unnecessary on lines 16, 19, and 21.

Although using Hound is optional, it is encouraged, and I've found it very helpful.  They may not want you to worry about it for this project.  :)

https://github.com/thoughtbot/hound